data/mods/gen6/random-teams.ts

```diff
@@ -961,26 +977,26 @@ export class RandomGen6Teams extends RandomGen7Teams {
 		Utils.sortBy(abilityNames, name => -this.dex.abilities.get(name).rating);
```

Comment:
I can't suggest changes to the line above this one but i'd suggest the same ability change i did in gen5/random-teams.ts